src/content/docs/alerts/create-alert/create-alert-condition/create-nrql-alert-conditions.mdx

```diff
@@ -716,19 +717,29 @@ Loss of signal settings include a time duration and two possible actions.
   * The loss of signal expiration time is independent of the threshold duration and triggers as soon as the timer expires.
   * The maximum expiration duration is 48 hours. This is helpful when monitoring for the execution of infrequent jobs. The minimum is 30 seconds, but we recommend using at least 3-5 minutes.
 * <DNT>**Loss of signal actions**</DNT>
-  Once a signal is considered lost, you can close open incidents, open new incidents, or both.
+  Once a signal is considered lost, you have a few options.
   * Close all current open incidents: This closes all open incidents that are related to a specific signal. It won't necessarily close all incidents for a condition. If you're alerting on an ephemeral service, or on a sporadic signal, you'll want to choose this action to ensure that incidents are closed properly. The GraphQL node name for this is ["closeViolationsOnExpiration](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/#loss-of-signal)"
   * Open new incidents: This will open a new incident when the signal is considered lost. These incidents will indicate that they are due to a loss of signal. Based on your incident preferences, this should trigger a notification. The graphQL node name for this is ["openViolationOnExpiration](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/#loss-of-signal)"
-  * When you enable both actions, we'll close all open incidents first, and then open a new incident for loss of signal.
+  * When you enable both of the above actions, we'll close all open incidents first, and then open a new incident for loss of signal.
+  * Do not open "lost signal" incident on expected termination: When a signal is expected to terminate, you can choose not to open a new incident. This is useful when you know that a signal will be lost at a certain time, and you don't want to open a new incident for that signal loss. The GraphQL node name for this is ["ignoreOnExpectedTermination](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/#loss-of-signal)"
+
+<Callout variant="important">
+  In order to prevent a loss of signal incident from opening when "Do not open "lost signal" incident on expected termination", the tag `termination: expected` must be added to the entity. This tag tells us the signal was expected to terminate. See how to add the tag [directly to the entity](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-tags).
```

Comment:
```suggestion
  In order to prevent a loss of signal incident from opening when "Do not open "lost signal" incident on expected termination", the tag `termination: expected` must be added to the entity. This tag tells us the signal was expected to terminate. See [how to add the tag directly to the entity](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-tags).
```